netci.groovy

```diff
@@ -365,6 +365,9 @@ def static getJobName(def configuration, def architecture, def os, def scenario,
         case 'x86':
             baseName = architecture.toLowerCase() + '_' + configuration.toLowerCase() + '_' + os.toLowerCase()
             break
+        case 'x86lb':
+            baseName = 'x86_lb_' + configuration.toLowerCase() + '_' + os.toLowerCase()
+            break
```

Comment:
why use 'x86_lb_' instead of 'x86lb'? If you used 'x86lb' you could just add `case 'x86lb':` next to `case 'x86'` and reuse that code.